interp/src/test.rs

```diff
@@ -0,0 +1,924 @@
+#[cfg(test)]
```

Comment:
Are these tests run by anything?

Reply:
cargo test? Or are you asking something else?

Reply:
Don't mind this comment. I was asking if these tests are executed by the CI.